apps/src/feedback.js

```diff
@@ -883,13 +886,7 @@ FeedbackUtils.prototype.getGeneratedCodeElement_ = function(options) {
       options.generatedCodeDescription);
   var code = this.studioApp_.polishGeneratedCodeString(this.getGeneratedCodeString_());
 
-  var codeDiv = document.createElement('div');
-  codeDiv.innerHTML = require('./templates/code.html.ejs')({
-    message: infoMessage,
-    code: code
-  });
-
-  return codeDiv;
+  return (<Code message={infoMessage} code={code} />);
 };
```

Comment:
We can, though my own preference would to mostly reserve that for cases where we have more dynamic properties. In this case, I feel like I'd rather not have to go peek into getGeneratedCodeProperties to know what I was setting on Code. In a world with ES6 (actually, technically this might need ES7 for object destructing) we could do:
```
var { message, code } = this.getGeneratedCodeProperites_(options);
return <Code message={message} code={code}/>;